src/index.tsx

```diff
@@ -677,6 +706,8 @@ const Gallery = <T extends any>({
   style,
   keyExtractor,
   containerDimensions,
+  onScaleValueChanged,
+  scaleRange,
```

Comment:
Please, rename
`onScaleValueChanged` => `onScaleChange`
`scaleRange` => `onScaleChangeRange`

Reply:
`onScaleValueChanged` is based `scaleRange` prop. `onScaleChangeRange` would indicate, that this is callback, but it is not true. It is just an object with `start` and `end` field. 

I will rename  onScaleValueChanged => onScaleChange

Reply:
I'm trying to follow FlatList prop names, e.g: `onEndReached`, `onEndReachedThreshold`
So I guess `onScaleChangeRange` will indicate that it's connected to `onScaleChange`

Reply:
`scaleRange` can be understood as range of possible scale of gesture

Reply:
@pavelbabenko thank you for an explanation, I will rename it.